src/related.tsx

```diff
@@ -98,14 +98,14 @@ class Related extends KalturaPlayer.core.BasePlugin {
       label: 'kaltura-related-overlay-next',
       presets: PRESETS,
       area: 'OverlayPlaybackControls',
-      get: () => <Next {...nextProps} />
+      get: () => <Next {...{...nextProps, showPreview: false}} />
```

Comment:
```suggestion
      get: () => <Next {...nextProps, showPreview: false} />
```

Reply:
doesn't work
the component expects {...props}